airflow/providers/amazon/aws/example_dags/example_redshift_data_execute_sql.py

```diff
@@ -71,10 +68,8 @@ def output_results_fn(id):
         poll_interval=POLL_INTERVAL,
         await_result=True,
     )
+    # [END howto_redshift_data]
 
-    # Using a task-decorated function to output the list of tables in a Redshift cluster
-    output_results_fn(redshift_query.output)
-
+    task_output = output_query_results(task_query.output)
 
-example_redshift_data_dag = example_redshift_data()
-# [END howto_redshift_data]
+    chain(task_query, task_output)
```

Comment:
This is redundant. The dependencies between these tasks should be handle implicitly via the TaskFlow API in `output_query_results(task_query.output)`

Reply:
Good to know! Thanks :)